src/System.Data.SqlClient/src/System/Data/SqlClient/TdsParserSafeHandles.cs

```diff
@@ -206,9 +206,9 @@ internal uint Status
         }
     }
 
-    internal sealed class SNIPacket : SafeHandle
+    internal sealed class SNIPacketHandle : SafeHandle
```

Comment:
Please don't change the type names. Helps us port changes more easily in case a change to Framework code is needed and vice versa.